config/routes.rb

```diff
@@ -16,8 +16,12 @@
   resource :cart, only: [:show, :destroy, :create]
   patch '/items/:id' => 'carts#update'
 
+  post '/checkout' => 'orders#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.